protocol.md

```diff
@@ -2204,6 +2204,11 @@ interface WorkspaceSymbolParams {
 	 * A non-empty query string
 	 */
 	query: string;
+
+	/**
+	 * Identifies the textDocument that is currently active.
+	 */
+	textDocument?: TextDocumentIdentifier;
```

Comment:
I believe we normally prefix non upstream extensions with an `x`, so it is clear what is unique to SG.